source/getting-started/concepts-terminology.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: "Concepts and terminology"
+description: "User explanation of Home Assistant basics"
+---
+Now you're in Home Assistant, let's go over important parts of Home Assistant and what they do.
+
+## Dashboards
+
+Customizable pages to display information connected to and available in Home Assistant. By default there are two Dashboards, Overview and Energy. The Overview dashboard is the first thing you see after the [onboarding process](/getting-started/onboarding). More details can be found [here](/dashboards).
+
+![Dashboard](/images/getting-started/lovelace.png)
+## Integrations
+
+Software that allows Home Assistant to connect to other software and platforms. For example, the Philips Hue integration allows Home Assistant to talk to the Hue Bridge and read data such as the state of your lights through to controlling the lights themselves.
+
+![Integrations](/images/getting-started/integrations.png)
+
+A full list of compatible integrations can be found [here](/integrations).
+
+Once an integration has been added, the hardware and/or data are represented in Home Assistant as [devices and entities](#devices--entities).
+
+## Devices & Entities
+
+Devices are a logical grouping for one or more entities. A device may represent a physical device and the device may have one or more sensors which appear as entities associated to the device. For example, a motion sensor is represented as a device and the motion may provide motion detection, temperature and light levels as entities. Entities have states such as on when motion is detected and off when there is no motion.
+
+Devices and entities can be used in dashboards to [automations](#automations).
```

Comment:
Not sure I understand this. Do you mean something like _Devices and entities are used in many areas in Home Assistant, e.g. in dashboards and automations_ ?

Reply:
@c0ffeeca7 good point. I will amend this and expand it a bit.

Reply:
@c0ffeeca7 how about this:
```
Devices and entities are used throughout Home Assistant. To name a few examples:
* Dashboards can show a state of an entity like if a light bulb is on or off as well as buttons that interact with devices like turning a light bulb on or off.
* An automation can be triggered from a state change on an entity e.g. a light turning on and control another device or entity.
* A pre defined setting for light device saved as a scene.